schema.md

```diff
@@ -0,0 +1,136 @@
+# Sparkle Motion schema (WIP)
+
+All record types have `createdAt` (and maybe `modifiedAt`?).
+
+### User
+
+```js
+{
+  "id": ___,
+  "username": "xian",
+  "password": "...",      // salted hash of pw
+  "name": "Full Name",
+  "priority": 123.4,      // e.g. for locking out other users during a performance
```

Comment:
How is priority determined? 

Is this a thing we manually delegate and assign ahead of time? Or is this generated based on who is currently connected or logged in to Pinky?

Reply:
I was thinking it's just arbitrarily assigned by an admin; presumably us, trained folks, and people running parties would get higher numbers than guests.